renta_facil_colima/src/View/components/DatePicker.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+import { View, Text, TouchableOpacity, StyleSheet } from 'react-native';
+import DatePicker from 'react-native-date-picker';
+import styles from '../../styles/DataPickerStyle';
+
+const DateSelector = () => {
+  const [selectedDate, setSelectedDate] = useState(new Date());
+  const [isDatePickerVisible, setIsDatePickerVisible] = useState(false);
+
+  const toggleDatePicker = () => {
+    setIsDatePickerVisible(!isDatePickerVisible);
+  };
+
+  const handleDateChange = (date) => {
+    setSelectedDate(date);
+    console.log('Fecha seleccionada:', date);
+    toggleDatePicker();
+  };
```

Comment:
This data is not going anywhere, you are not sending it to any parent or anything else